### PR TITLE
Journal formatting changes

### PR DESCRIPTION
**Pull request by jjc2718**

### CODE REVIEW

File: content/02.main-text.md [hunk 1]
```diff
@@ -285,31 +157,136 @@ Similarly, mutation calling in CCLE cell lines is limited by the lack of a match
...
+Software developed and used in this manuscript is available on GitHub at <https://github.com/greenelab/pancancer-evaluation/>, and on Zenodo at <https://doi.org/10.5281/zenodo.13960412>.
 The scripts used to download and preprocess the datasets for this study are available at <https://github.com/greenelab/pancancer-evaluation/tree/master/00_process_data>.
 Scripts for TCGA <-> CCLE comparisons (Figures 2 and 3) and neural network experiments (Figure 5) are available in the <https://github.com/greenelab/pancancer-evaluation/tree/master/08_cell_line_prediction> directory.
 Scripts for TCGA cancer type comparisons (Figure 4) are available in the <https://github.com/greenelab/pancancer-evaluation/tree/master/02_cancer_type_classification> directory.
 All scripts are available under the open-source BSD 3-clause license.
 
 This manuscript was written using Manubot [@doi:10.1371/journal.pcbi.1007128] and is available on GitHub at <https://github.com/greenelab/generalization-manuscript> under the CC0-1.0 license.
+
+## Acknowledgments
+
```
Comment (jjc2718) [inline content/02.main-text.md:282]: @cgreene Could you add funding/grant number information in this section as a proposed change, when you get a chance?

File: content/02.main-text.md [hunk 2]
```diff
@@ -285,31 +157,136 @@ Similarly, mutation calling in CCLE cell lines is limited by the lack of a match
...
 This research was supported in part by the University of Pittsburgh Center for Research Computing through the resources provided. Specifically, this work used the HTC cluster, which is supported by NIH award number S10OD028483.
```
Comment (cgreene) [inline content/02.main-text.md:283]:
```suggestion
This research was supported in part by grants from the National Institutes of Health (R01 CA237170 and R01 HG010067).
This research was supported in part by the University of Pittsburgh Center for Research Computing through the resources provided. Specifically, this work used the HTC cluster, which is supported by NIH award number S10OD028483.
```

**Review by cgreene — APPROVED** [review]

File: content/02.main-text.md (hunk 1 above)
Comment (cgreene) [reply content/02.main-text.md:282]: Got it - want me to merge & then submit?

Comment (jjc2718) [reply content/02.main-text.md:282]: Sure, if you don't mind! They also want us to submit the figures separately, as PDFs - I'll email them to you.

**Pull request merged**